frontend/elements/src/module/_groups/cards/play/card/card.ts

```diff
@@ -95,6 +95,10 @@ export class _ extends LitElement {
     @property({ type: Boolean, reflect: true })
     flipped: boolean = false;
 
+    // whether or not showing front vs. back
+    @property({ type: Boolean })
+    eventOnFlipped: boolean = false;
```

Comment:
Using this property on the element itself seemed liked the easiest way without overhauling the `CardOptions` and `Current` structs so that data in those structs could live for `'static`.

The purpose of this property is to prevent the event being dispatched in some cases where we wouldn't want to play the audio effect.